src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj

```diff
@@ -7,7 +7,7 @@
         <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
         <DefineConstants>$(DefineConstants);BUILDTASK;XAMLX_CECIL_INTERNAL;XAMLX_INTERNAL</DefineConstants>
         <CopyLocalLockFileAssemblies Condition="$(TargetFramework) == 'netstandard2.0'">true</CopyLocalLockFileAssemblies>
-        <NoWarn>NU1605</NoWarn>
+        <NoWarn>NU1605;NU5128</NoWarn>
```

Comment:
It is safe suppress thi warning?

Reply:
today I was able to find what the notice NU5128 depends on. It depended on the version of sourcelink.create.commandline